### PR TITLE
Add support for Python 3.10

### PR DESCRIPTION
**Pull request by joshuadavidthomas**
- Adds 3.10 to the strategy matrix in the `ci.yml` workflow with [passing tests](https://github.com/joshuadavidthomas/python-phonenumbers/actions)
- Updates trove classifiers to include 3.10 as a supported version

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-18.04
     strategy:
       matrix:
-        python-version: [2.7, 3.3, 3.4, 3.5, 3.6, 3.7, 3.8, 3.9, pypy2, pypy3]
+        python-version: [2.7, 3.3, 3.4, 3.5, 3.6, 3.7, 3.8, 3.9, "3.10", pypy2, pypy3]
```
Comment (daviddrysdale) [inline .github/workflows/ci.yml:10]: nit: slightly inconsistent formatting &ndash; does this need to be quoted?

**Review by daviddrysdale — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (joshuadavidthomas) [reply .github/workflows/ci.yml:10]: Yes, otherwise it gets read as 3.1 because of a quirk of YAML interpreting it as a float instead of string.

Comment (joshuadavidthomas) [reply .github/workflows/ci.yml:10]: https://twitter.com/webology/status/1445593297750147074

Comment (joshuadavidthomas) [reply .github/workflows/ci.yml:10]: https://dev.to/hugovk/the-python-3-1-problem-85g

Comment (daviddrysdale) [reply .github/workflows/ci.yml:10]: Thanks for confirming, I'd guessed it might be something along those lines.

**Pull request merged**